File: lec_05/phone_book.cpp
#include <iostream>
#include <unordered_map>
#include <string>

using namespace std;

unordered_map<string, int> phoneBook = {
        {"Jenny", 876453},
        {"Emergency", 911}
    };

void check_phone(const string &name) {
    unordered_map<string, int> :: iterator it;
    it = phoneBook.find(name); 
        cout << it->second << '\n';
}


int main() {
    string name;
    cin >> name;
    check_phone(name);
}

File: lec_05/phone_book.py
phone_book = {
    "Jenny": 876453,
    "Emergency": 911
}


def check_phone(name):
    if phone_book.get(name):
        print(phone_book.get(name))
    else:
        print("User not found")
        pass

check_phone("Alice")
check_phone("Jenny")
check_phone("Emergency")